src/constants.js

```diff
@@ -121,6 +121,7 @@ export const CLIENT_NAME_CONTENT_UPLOADER = 'ContentUploader';
 export const CLIENT_NAME_CONTENT_EXPLORER = 'ContentExplorer';
 export const CLIENT_NAME_CONTENT_PREVIEW = 'ContentPreview';
 export const CLIENT_NAME_CONTENT_SIDEBAR = 'ContentSidebar';
+export const BOX_NOTE_EXTENSION = 'boxnote';
```

Comment:
Probably FILE_EXTENSION_BOX_NOTE.
Just incase we add more, easier to search with generic prefix

Reply:
Maybe also add in its own section /*---*/